rust/automerge/src/transaction/manual_transaction.rs

```diff
@@ -39,6 +39,10 @@ impl<'a, Obs: observation::Observation> Transaction<'a, Obs> {
             observation: Some(obs),
         }
     }
+
+    pub fn hash_for_opid(&self, opid: &ExId) -> Option<ChangeHash> {
```

Comment:
As you say in the comments, this feels like it might be confusing. Could we add some docs to the effect that if you lookup an opid you created in this transaction you'll get `None`?

I suppose we could even detect that it's an opid created in this transaction and have a return type that reflects that. Something like:

```rust
enum HashInTx {
    ThisTx,
    OlderChange(ChangeHash)
}
```

Reply:
I've added the docs. Given the differences when calling it on a Automerge / Transaction vs an AutoCommit I think keeping the types the same is easiest for now.